boards/st/stm32h747i_disco/stm32h747i_disco_stm32h747xx_m7.dts

```diff
@@ -64,13 +64,37 @@
 		led1 = &orange_led_2;
 		sw0 = &wake_up;
 	};
+
+	dsi_lcd_qsh_030: connector_dsi_lcd {
+		compatible = "st,dsi-lcd-qsh-030";
```

Comment:
Please add a dedicated yaml binding. See dts/bindings/gpio/arduino-header-r3.yaml as example

Reply:
@erwango there is already one `dts/bindings/gpio/st,dsi-lcd-qsh-030.yaml`

Reply:
Ok, thanks, my bad.